gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
```

Comment:
Thanks for including the patlet!  I think it would be good to include a sentence about the problem that this pattern is trying to solve and also reduce the description of the solution to just 1-2 sentences.